pages/blog/_posts/open communication.md

```diff
@@ -0,0 +1,38 @@
+# What are your weekend plans?
```

Comment:
This section is missing a paragraph. It looks like the paragraph belonging to `Open communication as a tool to break down barriers` would fit well here.

Reply:
it is the headline. Might only need it in the File name???